Plugins/GitUIPluginInterfaces/IProcess.cs

```diff
@@ -10,6 +10,8 @@ namespace GitUIPluginInterfaces
     /// </remarks>
     public interface IProcess : IDisposable
     {
+        const int DefaultExitTimeoutMilliseconds = 500;
```

Comment:
Make it `ushort`?

Reply:
Why? This is given to `Process.WaitForProcess(int milliseconds)`.